scripts/find-low-dpi-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Created by Sam Gleske (@samrocketman on GitHub)
+# MIT licensed
+# Sat Aug 27 13:35:25 EDT 2022
+# Pop!_OS 18.04 LTS
+# Linux 5.4.0-113-generic x86_64
+# GNU bash, version 4.4.20(1)-release (x86_64-pc-linux-gnu)
+# find (GNU findutils) 4.7.0-git
+# xargs (GNU findutils) 4.7.0-git
+# git version 2.17.1
```

Comment:
I don't think you need all this?

Reply:
I add the development environment in which scripts were created since tools change

Reply:
More specifically someone running from Mac (BSD tools) should understand this was developed from Linux (GNU or findutils tools).  There may be changes necessary for a Mac user.

While the version on Mac bash is very old (v3 I think) I am using bash features available to it.